### DISCUSSION TITLE
dolphinscheduler3.1.0 password not available change

### DISCUSSION THREAD

**Discussion by trustTiMo** [apache/dolphinscheduler#12251] [General]
dolphinscheduler3.1.0 password not available change
![image](https://user-images.githubusercontent.com/78303733/193727138-b39f20ae-7d2c-41a5-8efa-670614d08d9c.png)

**Comment by lyn916** [comment]
![image](https://user-images.githubusercontent.com/39552917/194709946-247a4612-94a7-477d-af16-3780ccdb83aa.png)
哪里的bug，解决一下啊。同样的问题
